config/ergw-c-node.config

```diff
@@ -107,6 +90,16 @@
 	   }]
 	 },
 
+	 {path_management,
```

Comment:
@fholzhauser we need a description of how to tune the settings in a gtp-proxy setup for the NEWS.md to describe also the known-issue.